src/js/bandwidth_test.js

```diff
@@ -176,24 +177,55 @@ VideoBandwidthTest.prototype = {
       this.completed();
     } else {
       this.test.setProgress((now - this.startTime) * 100 / this.durationMs);
-      this.call.pc1.getStats(this.gotStats.bind(this));
+      this.call.pc1.getStats(this.localStream)
+      .then(function(response) {
```

Comment:
You don't need to create a function here, just do the bind:
 .then(this.gotStats.bind(this).catch(...)